src/chart/cartesian.js

```diff
@@ -177,36 +177,30 @@ export default function(xScale, yScale) {
         });
     };
 
-    function rebindScale(scale, prefix) {
-        var scaleExclusions = [
-            /range\w*/,   // the scale range is set via the component layout
-            /tickFormat/  // use axis.tickFormat instead (only present on linear scales)
-        ];
-
-        // The scale ticks method is a stateless method that returns (roughly) the number of ticks
-        // requested. This is subtley different from teh axis ticks methods that simple stores the given arguments
-        // for invocation of the scale method at some point in the future.
-        // Here we expose the underling scale ticks method in case the user want to generate their own ticks.
-        if (!isOrdinal(scale)) {
-            scaleExclusions.push('ticks');
-            var mappings = {};
-            mappings[prefix + 'ScaleTicks'] = 'ticks';
-            rebind(cartesian, scale, mappings);
-        }
-
-        rebindAll(cartesian, scale, prefix, scaleExclusions);
+    var scaleExclusions = exclude(
+        /range\w*/,   // the scale range is set via the component layout
+        /tickFormat/  // use axis.tickFormat instead (only present on linear scales)
+    );
+    rebindAll(cartesian, xScale, scaleExclusions, prefix('x'));
+    rebindAll(cartesian, yScale, scaleExclusions, prefix('y'));
```

Comment:
From reviewing the code, I think you're right, the net effect is the same. However, it would be all to easy to move the order of the methods around and change this behaviour.